master/docs/relnotes/index.rst

```diff
@@ -44,7 +46,7 @@ Deprecations, Removals, and Non-Compatible Changes
 * The buildmaster now requires at least Twisted-14.0.1.
 
 * GitLabStatus was present in buildbot 0.8.12 release, but is no more in 0.9.0 (:bug:`3286`).
```

Comment:
actually it wasn't as we saw yesterday on IRC. You can drop this remark.
You can say:

* new :bb:reporter:`GitLabStatusPush` to report builds results to gitlab.